operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -310,15 +309,15 @@ func TestInstallPackagePath(t *testing.T) {
 
 // This test enforces that objects that reference other objects do so properly, such as Service selecting deployment
 func TestConfigSelectors(t *testing.T) {
-	got, err := runManifestGenerate([]string{}, "", true)
+	got, err := runManifestGenerate([]string{}, "", false)
```

Comment:
This test must use the real config files, not the fake snapshot

Reply:
i agree that this test should use real config files, i'll change this. for the other cases below, i believe we should be using the snapshot (see individual comments)

Reply:
done